src/SciGlassCalorimeterPanda_geo.cpp

```diff
@@ -257,7 +257,15 @@ static Ref_t create_detector(Detector& desc, xml::Handle_t handle, SensitiveDete
   return det;
 }
 
+static Ref_t create_detector_deprecated(Detector& desc, xml::Handle_t handle, SensitiveDetector sens) {
+  printout(WARNING, "SciGlassCalorimeter", "#########################################################################################");
+  printout(WARNING, "SciGlassCalorimeter", "\"SciGlassCalorimeter\" is a deprecated name, please use \"SciGlassCalorimeterPanda\" instead");
+  printout(WARNING, "SciGlassCalorimeter", "#########################################################################################");
+  return create_detector(desc, handle, sens);
+}
+
 #ifdef EPIC_ECCE_LEGACY_COMPAT
-DECLARE_DETELEMENT(ecce_SciGlassCalorimeter, create_detector)
+DECLARE_DETELEMENT(ecce_SciGlassCalorimeter, create_detector_deprecated)
 #endif
-DECLARE_DETELEMENT(epic_SciGlassCalorimeter, create_detector)
+DECLARE_DETELEMENT(epic_SciGlassCalorimeter, create_detector_deprecated)
+DECLARE_DETELEMENT(epic_SciGlassCalorimeterPanda, create_detector)
```

Comment:
In the past we placed deprecated geometry plugins in `src/deprecated` to make this more manifest as well.

Reply:
I'm not deprecating the Panda design yet, only the unspecific detector type `epic_SciGlassCalorimeter`. The `epic_SciGlassCalorimeterPanda` is still the default.

Reply:
Ah, I see now.